yarn-project/noir-contracts/src/contracts/counter_contract/src/main.nr

```diff
@@ -16,17 +16,15 @@ contract Counter {
             },
     };
     use dep::easy_private_state::easy_private_state::EasyPrivateUint;
-// docs:end:imports
-
-// docs:start:storage_struct
 
+    // docs:end:imports
+    // docs:start:storage_struct
```

Comment:
Do you know if this will affect docs codegen?

Reply:
Looks like it removed some of the docs stuff though 💀 docs now building because it cannot find ends